### ISSUE TITLE
Hanami serialization example is wrong

### ISSUE THREAD

**Issue by luciusgone** [jsonapi-rb/jsonapi-rb.github.io#14]
I suppose there is an example that is wrong about hanami framework.
the file is located at source/guides/serialization/rendering.html.md.

The example says:
```ruby
module API::Controllers::Posts
  include API::Action
  include JSONAPI::Hanami::Action

  expose :url_helpers

  class Create
    # ...
    @url_helpers = routes  # Will be available inside serializable resources.

    self.data = posts
    self.include = [:author, comments: [:author]]
    self.fields  = { users: [:name, :email],
                     posts: [:title, :content] }
  end
end
```
It should be like this:
```ruby
module API::Controllers::Posts
  class Create
    include API::Action
    include JSONAPI::Hanami::Action

    expose :url_helpers

    def call(params)
      #...
      @url_helpers = routes  # Will be available inside serializable resources.

      self.data = posts
      self.include = [:author, comments: [:author]]
      self.fields  = { users: [:name, :email],
                       posts: [:title, :content] }
    end
  end
end
```

**Comment by beauby** [comment]
Hi @luciusgone – you are right indeed. Would you mind opening a PR to fix it?

**Comment by beauby** [comment]
Fixed by #15.
